.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for submitting this issue.'
+        pr-message: 'Thank you for your contribution to this repository. We will be reviewing your PR. In the meantime please sign the CLA below.'
```

Comment:
This message might come before or after the CLA message presents itself. We might want to reword this to `In the meantime, don't forget to sign the CLA.` or something of that nature.

Reply:
> ohhh got it, I'll change that now! :)